### DISCUSSION TITLE
how to block drawing?

### DISCUSSION THREAD

**Discussion by doxomo** [vinothpandian/react-sketch-canvas#110] [Q&A] [answered]
How can I block drawing? Make it view-only mode

**Comment by vinothpandian** [comment]
Right now there is no option. But if you import the Canvas instead of ReactSketchCanvas you can easily solve this by using the handlers

**Comment by dninomiya** [comment, marked as answer]
I solved with CSS pointer-event.

```tsx
<button onClick={() => setReadOnly(true)}>read only</button>
<div className={!readOnly && 'pointer-events-none'}>
  <ReactSketchCanvas />
</div>
```

(with Tailwind CSS)

**Reply by vinothpandian** [reply]
Related Issue: https://github.com/vinothpandian/react-sketch-canvas/issues/125

**Comment by vinothpandian** [comment]
From [release v7.0.0.next.4](https://github.com/vinothpandian/react-sketch-canvas/releases/tag/react-sketch-canvas%407.0.0-next.4), you can use `readOnly` prop to disable drawing
